File: pyspark/SparkSessionEx.py
from pyspark.sql import SparkSession
if __name__ == "__main__":
    spark = SparkSession.builder.appName("PySpark Spark Session Example").master("local[*]").getOrCreate()

    num_range = spark.range(0, 10)

    num_range.show()

    spark.stop()

File: spark-scala/src/main/scala/SparkSessionEx.scala
import org.apache.spark.sql.SparkSession

object SparkSessionEx {
  def main(args: Array[String]): Unit = {
    val spark = SparkSession.builder().appName("Spark Example").getOrCreate()

    val range = spark.range(0, 10)

    range.foreach()

  }
}